### ISSUE TITLE
Some files not generated by the translation system for Georgian language (ka)

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/mrconfig#80]
### Issue
The translation framework scripts do not create any file in the following folders of the SVN screenreadertranslations (SRT) repo:
* `ka\changes-newRevisions`
* `ka\symbols-newRevisions`
* `ka\userGuide-newRevisions`

### Context
Issue discussed initially [on the translation mailing-list](https://groups.io/g/nvda-translations/topic/enable_all_symbol_revisions/92686819?p=,,,100,0,0,0::recentpostdate/sticky,,,100,2,0,92686819,previd%3D1659095054626389849,nextid%3D1626473572385316354&previd=1659095054626389849&nextid=1626473572385316354), where Georgian translators need the symbol revisions.

For now the Georgian translators want to translate/update the interface (nvda.po) and the `symbols.dic` file. The character description is already translated. In the future, they may translate the user guide and maybe the change log.

Thus for now, the updates in the `symbols-newRevisions` folder are needed.

### Additional questions
* Can the translation framework scripts work correctly  even if `changes.t2t` and `userGuide.t2t` are not present in SVN? If not, this should be documented in the translation wikis.
* When the translators want to translate the userGuide, it would be better if the `userGuide-newRevisions` folder already contains files; this way they can take the file from the last revision and just translate it.

### Note
I could have given a try at looking to the scripts and adding the missing files (newRevisions, t2t) as well as modifying the `settings` file to see if it solves the problem. But I have to wait for a week before the scripts are triggered again.

So I prefer NVAccess to do it, since they can check the logs of the scripts and trigger them to check immediately the result.

@feerrenrut, could you have a look please?

**Comment by DraganRatkovich** [comment]
Hi,

Thanks @CyrilleB79 for opening the issue. I'm Shaban from the translation mailing list.

Looking forward to more details on this issue.

**Comment by DraganRatkovich** [comment]
Hello,
@CyrilleB79 @feerrenrut Is there any progress on this issue?

**Comment by seanbudd** [comment]
From what I understand from reading the thread, it appears that several missing files need to be created in [`/srt/ka`](http://subversion.assembla.com/svn/screenReaderTranslations/ka/).
For example, compare to [`/srt/es`](http://subversion.assembla.com/svn/screenReaderTranslations/es/).

It is not immediately clear to me what is preventing translators from pushing the necessary files to the repository, and what the best practice should be for adding these files if this is not possible.
Further investigation is needed, to answer this and your additional questions.

**Comment by DraganRatkovich** [comment]
Hello,
@seanbudd In any case, I will focus only on translating the interface. The Georgian language has long been abandoned, so there may be errors in the process of creating a file by the system. As soon as something changes regarding other translatable files, I will consider translating them.

**Comment by DraganRatkovich** [comment]
Hi,
@seanbudd Files are not created for a long time. Changes, symbols, user manual files, etc. So I don't know how to add them back to the Georgian folder. In addition, every time when new revisions of symbols and changes appear in folders in other languages, in this case the folder in Georgian remains empty. In any case, I will focus only on translating the interface. The Georgian language has long been abandoned, so there may be errors in the process of creating a file by the system. As soon as something changes regarding other translatable files, I will consider translating them.

**Comment by CyrilleB79** [comment]
I have added the missing files and updated the settings file with the last hash info.
Today, the first merge to beta of 2022.4 has been pushed in SRT repo and the new revision have been generated, at least for change log and user guide. Regarding symbols, there are no new symbols for 2022.4, so no new revision.

Thus this issue can now be closed. We may reopen it if something does not work.

Additional note: There is also another issue with 'ka' translation: the t2t files for change log and user guide (English) have been added by mistake in SVN SRT repo, and then have landed in beta branch of NVDA. They need to be removed since no translation of these files is planned for now. This separate request is tracked in https://github.com/nvaccess/nvda/pull/14252.
